.travis.yml

```diff
@@ -49,13 +49,14 @@ otp_release:
         - 18.3
 env:
         - PRESET=internal_mnesia DB=mnesia REL_CONFIG=with-all
+ken_disabled_upstream_env:
```

Comment:
Re-enable all the disabled presets for mainstream repo. I understand in your setup you need only one preset. Here in the main repo we need to test all the configurations.